packages/docker-node/src/types/index.ts

```diff
@@ -1,5 +1,13 @@
+import { CellDep } from '@ckb-lumos/lumos'
+
 export interface DockerNodeStartOptions {
   port: string
   detached?: boolean
   genesisAccountArgs?: string[]
 }
+
+export interface InfraScript {
+  name: string
+  path: string
+  cellDep: { cellDeps?: CellDep[] } & CellDep
```

Comment:
The type declaration is hard to understand, it will be 

```typescript
{
  cellDeps: CellDep[],
  ...CellDep,
}
```

Reply:
And only `cellDep: CellDep` is used at https://github.com/ckb-js/kuai/pull/257/files#diff-82a2e05889db2d2dbb18f524baf5085533ddf8c66a15de72075b77e2d2fc00e6R104

What's the use cases of `CellDeps` and `...CellDep`

Reply:
> And only `cellDep: CellDep` is used at https://github.com/ckb-js/kuai/pull/257/files#diff-82a2e05889db2d2dbb18f524baf5085533ddf8c66a15de72075b77e2d2fc00e6R104
> 
> What's the use cases of `CellDeps` and `...CellDep`

`...CellDep` is used to recording the out point information of the script which is deployed, and the `cellDeps` is used to providing the dependencies of the script which will be deployed.

Reply:
Seems that `CellDep` can be treated as `script's outPoint` or something. Putting `CellDep` and `CellDeps` together is hard to understand their relationship.

Reply:
Will we update this type declaration?

Reply:
> Will we update this type declaration?

Yes, I will push the commit later after test